src/components/intro/style.scss

```diff
@@ -170,6 +170,26 @@
   --color: #fff;
 }
 
+.install-button {
+  composes: unbutton from '../../lib/util.scss';
+
+  &:hover,
+  &:focus {
+    background: #f5f5f5;
+  }
+
+  background: #fff;
+  border: 1px solid #e8e8e8;
+
+  margin-top: 1em;
+
+  align-items: center;
+  padding: 14px;
+  font-size: 1.3rem;
+
+  display: none;
+}
```

Comment:
```suggestion
.install-button {
  composes: unbutton from '../../lib/util.scss';

  &:hover,
  &:focus {
    background: #f5f5f5;
  }

  background: #fff;
  border: 1px solid #e8e8e8;

  align-items: center;
  padding: 14px;
  font-size: 1.3rem;

  position: absolute;
  top: 1rem;
  right: 1rem;
  display: none;
  opacity: 0;

  animation: fade-in .3s linear 1s forwards;
}

@keyframes fade-in {
	to {opacity: 1;}
}
```

This places the button in the top right-hand corner with a quick fade-in. Because the `beforeinstallprompt` event can be quite delayed, the previous placement can make the button plop into view and be quite jarring and also cause a layout shift.

<details>

![Screenshot 2020-06-24 at 13 13 16](https://user-images.githubusercontent.com/234957/85554372-bc237000-b61c-11ea-8cbf-e34c276f086c.png)

</details>

Not sure this is what we want to ship, but just something to start the convo.

Reply:
I'm OK with this design, I primarily wanted to get something in that would have both an in-app promo, plus the browser promo. Potentially we can A/B test different locations later.